Gopkg.toml

```diff
@@ -130,6 +130,10 @@ ignored = [
   name = "github.com/miekg/dns"
   version = "1.0.15"
 
+[[constraint]]
```

Comment:
I didn't include it in here since I didn't see Ginkgo / Gomega as well (golang/mock will also just be a testing dependency). Are we missing them as well or did we leave them out on purpose?

Reply:
Good question... @msohn can you clarify please? Do you want us to add the licenses of all dependencies that are only used for tests as well to the LICENSE.md?

Reply:
I forwarded this question to our legal team and will come back with their advise